rrfs-test/IODA/yaml/bufr_ncep_satmar.yaml

```diff
@@ -0,0 +1,137 @@
+          height: 
+            query: "*/ALTPE"
+
+          # ObsValue
+          brightnessTemperature:
+            query: "*/TMBRST"
+          windSpeed:
+            query: "*/WSPA"
+          heightOfWaves: 
+            query: "*/HOWV"
+          windEastward:
+            query: "*/UMWV"
+          windNorthward:
+            query: "*/VWMV"    
+          
+          # QualityMarker 
+          #windAlongRadialLineQM: 
+          #query: "*/QCRW"
+
+    ioda:
+      backend: netcdf
+      obsdataout: "/work2/noaa/da/pkumar/ORION/RDASApp/urma.t00z.satmar.tm00.nc"
+      #obsdataout: "./urma.t00z.satmar.tm00.nc"
+
+      variables:
+        # MetaData
+        - name: "MetaData/dateTime"
+          coordinates: "longitude latitude"
+          source: variables/timestamp
+          longName: "Datetime"
+          units: "seconds since 1970-01-01T00:00:00Z"
+
+        - name: "MetaData/dataReceiptTime"
+          coordinates: "longitude latitude"
+          source: variables/receipttime
+          longName: "Data Receipt Time"
+          units: "seconds since 1970-01-01T00:00:00Z"
+
+        - name: "MetaData/latitude"
+          coordinates: "longitude latitude"
+          source: variables/latitude
+          longName: "Latitude"
+          units: "degree_north"
+          range: [-90, 90]
+
+        - name: "MetaData/longitude"
+          coordinates: "longitude latitude"
+          source: variables/longitude
+          longName: "Longitude"
+          units: "degree_east"
+          range: [-180, 180]
+
+        - name: "MetaData/satelliteIdentifier"
+          coordinates: "longitude latitude"
+          source: variables/satelliteIdentifier
+          longName: "Satellite Identifier"
+
+        - name: "MetaData/height"
+          coordinates: "longitude latitude"
+          source: variables/height
+          longName: "Altitude (Platform to Ellipsoid)"
+          units: "m"
+
+        # ObsValue
+        - name: "ObsValue/brightnessTemperature"
+          coordinates: "longitude latitude Channel"
+          source: variables/brightnessTemperature
+          longName: "Brightness temperature"
+          units: "K"
+
+        - name: "ObsValue/windSpeed"
+          coordinates: "longitude latitude"
+          source: variables/windSpeed
+          longName: "Wind Speed from Altimeter"
+          units: "m s-1"
+
+        - name: "ObsValue/heightOfWaves"
+          coordinates: "longitude latitude"
+          source: variables/heightOfWaves
+          longName: "Height Of Waves"
+          units: "m"
+
+        - name: "ObsValue/windEastward"
+          coordinates: "longitude latitude"
+          source: variables/windEastward
+          longName: "Eastward Wind"
+          units: "m s-1"
+
+        - name: "ObsValue/windNorthward"
+          coordinates: "longitude latitude"
+          source: variables/windNorthward
+          longName: "Northward Wind"
+          units: "m s-1"
+
+        # QualityMarker
```

Comment:
Can these commented lines be removed?

Reply:
@MatthewMorris-NOAA thanks, removed all the commented lines.